doc/man3/EC_GROUP_copy.pod

```diff
@@ -79,12 +79,16 @@ n-1 where n is the B<order>. The B<order> multiplied by the B<cofactor> gives th
 
 EC_GROUP_get0_generator returns the generator for the identified B<group>.
 
-The functions EC_GROUP_get_order and EC_GROUP_get_cofactor populate the provided B<order> and B<cofactor> parameters
-with the respective order and cofactors for the B<group>.
+EC_GROUP_get_order() retrieves the order of B<group> and copies its value into
+B<order>.  It fails in case B<group> is not fully initialized (i.e., its order
+is not set or set to zero).
+
+EC_GROUP_get_cofactor populate the provided B<order> and B<cofactor> parameters
+with the order and cofactors for the B<group>.
```

Comment:
This can be improved: this function does not do anything about `order` and we should also highlight that it _copies_ the value (in contrast with `EC_GROUP_get0_cofactor()` that returns a `const BIGNUM*` pointer to the value stored inside the `EC_GROUP` structure).

My suggestion would be to reuse the same text used for `EC_GROUP_get_order()` above, adapting it, e.g.:

```pod
EC_GROUP_get_cofactor() retrieves the cofactor of B<group> and copies its value
into B<cofactor>. It fails in case  B<group> is not fully initialized or if the
cofactor is not set (or set to zero).
```